operations/boot_orchestration/Multi_tenancy_with_BOS.md

```diff
@@ -21,15 +21,38 @@ resources specific to that tenant.
 
 ### Components
 
-Component ownership is not directly tracked by BOS to avoid duplicating information stored in TAPMS and HSM. As a result
+Component ownership is not directly tracked by `BOS` to avoid duplicating information stored in `TAPMS` and HSM. As a result
 the system administrator will not see any tenant information in the component records and the components endpoint will function
-normally for system administrators. To determine which components belong to which tenant, the administrator will need to query TAPMS.
+normally for system administrators. To determine which components belong to which tenant, the administrator will need to query `TAPMS`.
 
 ## Sessions and session templates
 
 Both session and session template records now contain information on the tenant that owns them. System administrators can list
-all sessions and session templates across all tenants. However system administrators will not be able to create, retrieve,
+all sessions and session templates across all tenants. However, system administrators will not be able to create, retrieve,
 update, or delete specific records that belong to a tenant without authenticating to that specific tenant. This is
 because the resources are name-spaced to avoid naming collisions between tenants. The name-spacing means that there can
-be two or more records with the same name, and BOS does not know which resource is requested unless the tenant is
+be two or more records with the same name, and `BOS` does not know which resource is requested unless the tenant is
 specified. To access tenant owned resources, the system administrator will need to masquerade as that tenant.
+
+## BOS API Access with Tenancy
+
+`BOS` v2 API introspects request information pertaining to tenant information, if provided. If you have tenant information
```

Comment:
What does this sentence mean?
`BOS` v2 API introspects request information pertaining to tenant information, if provided.

I think you're saying the following which I would suggest could be a replacement for that sentence.
The `BOS` v2 API identifies the tenant based on information passed in the request. A tenant must always pass information identifying itself -- their tenant ID -- when making a request.

I think the additional following re-wording is also more clear and could replace the paragraphs up to the `Note:`.

The CrayCLI will supply the tenant ID as part of any requests made through it, provided that the tenant logged into CrayCLI via 'cray init'.

Alternately, when making requests to the Cray API directly, the user must supply the tenant ID as part of the request.

The tenant ID is provided as a header value in the request. It is of the form `Cray-Tenant-Name: <tenant name>`.